packages/admin-icons/icons/master unlock.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Should this be renamed to `master-unlock.svg`? It's the only file with a space...